Sprint2

Iraitz-Elorrieta opened the pull request:
Departamentuak eta Langileak erakutsi (XSL).  Departamentuen datuak eta langileen datuak erakutsi (XSL).

Iraitz-Elorrieta committed:
sprint2 amaitzen langileburua ateratzea falta da.

sprint2 amaitzen langileburua ateratzea falta da.

Iraitz-Elorrieta committed:
Langileak igota

Langileak igota

miraricasillas committed:
aldatzen xsl

Iraitz-Elorrieta committed:
Intranet Zaharra kontrastatzeko (php)

Intranet Zaharra kontrastatzeko (php)

Iraitz-Elorrieta committed:
MIMI Zaharra kontrastatzeko (php)

MIMI Zaharra kontrastatzeko (php)

miraricasillas committed:
langileak xsl (argazkiak + izena)

langileak xsl (argazkiak + izena)

miraricasillas committed:
sprint2 amaituta

sprint2 amaituta

miraricasillas committed:
Irudiak (departamentua)

Irudiak (departamentua)

miraricasillas committed:
2sprint amaituta

2sprint amaituta

miraricasillas committed:
Delete MIMI/css/langileak.css

miraricasillas committed:
Delete MIMI/css/departamentuak.css

Iraitz-Elorrieta committed:
Delete oldMIMI(php) directory

Iraitz-Elorrieta committed:
Footer conponduta

Footer conponduta

Iraitz-Elorrieta committed:
Departamentua izendu eta Langile bati eman

Departamentua izendu eta Langile bati eman

Iraitz-Elorrieta committed:
error_mensage ezabatu

error_mensage ezabatu

Nereee reviewed (changes requested):
Departamentuak ez dira moldagarriak, agian langileen erara antolatu ahal dituzue. 
Eta datuen zerrendan, bai langileenean zein departamentuarenean, argibide batzuk jarri, ez dago argi datu bakoitza zer den

Iraitz-Elorrieta committed:
Datuak identefikatu. Departamentuak responsive

Langile eta departamentuetan datuak identefikatu eta Departamentuak responsive egin.

Iraitz-Elorrieta commented:
Langile eta departamentuetan datuak identefikatuta daude eta departamentuen css arazoak conponduta.

Nereee reviewed (commented):
Responsive da. Baina finkatzerakoan zutabe kopurua 4-ra, beti, agindu honekin: grid-template-columns: repeat(4, minmax(min(150px, 100%), 0.2fr));
Kutxak oso txikiak ikusten dira.  Agian dinamikoagoa egin ahal duzue...
Zuen esanetara horrela utzi edo ez....

Iraitz-Elorrieta committed:
css aldaketak(3), dena handiago Index url aldaketa

css aldaketak (3) eta dena handiago. Index horrialdean url aldaketak.

Iraitz-Elorrieta committed:
2sprint amaituta Css guztia eta xsl bat kendu

2sprint amaituta Css guztia eta xsl bat kendu

Iraitz-Elorrieta committed:
Delete MIMI/datuak/xsl/datuak.xsl

Iraitz-Elorrieta commented:
Css- a responsive da guztiz

Iraitz-Elorrieta committed:
sprint3

sprint3

Iraitz-Elorrieta committed:
Sprint3

Sprint3

Nereee commented:
Oso ondo! errepasatu orri nagusian apurtutako estekaren bat dago eta.
